### ISSUE TITLE
history() returns dict when any of the symbols are invalid

### ISSUE THREAD

**Issue by AbrahamWisman** [dpguthrie/yahooquery#126]
**Describe the bug**
If any of the given symbols are invalid, like the outdated fb. The history() method will return a dict.
I am using python 3.10 and yahooquery 2.2.15.
What I dont understand is that this issue has been mentioned before and marked as fixed but it still exist?

**To Reproduce**
(https://yahooquery.dpguthrie.com/guide/ticker/historical/)

```
tickers = Ticker('fb aapl nflx', asynchronous=True)
df = tickers.history()
``` 
results in dict

```
tickers = Ticker('meta aapl nflx', asynchronous=True)
df = tickers.history()
``` 
results in df

**Expected behavior**
Ignore invalid symbols, maybe print a warning like yfinance does.
"FB: No data found, symbol may be delisted"

**Comment by maread99** [comment]
Seems the behavior was changed in #118 which has been merged but the changes haven't yet been released to PyPI.

You'll get the latest behavior If you install from the master branch - looks like `.history` now always return a `DataFrame` with invalid symbols silently ignored.

**Comment by dpguthrie** [comment]
This is resolved in 2.3.0
